packaging/nnstreamer.spec

```diff
@@ -394,6 +394,12 @@ Requires:	gstreamer-devel
 Development package for custom tensor operator developers (tensor_filter/custom).
 This contains corresponding header files and .pc pkgconfig file.
 
+%package devel-internal
+Summary:    Internal development package for sub-plugins and native APIs
+Requires:   devel = %{version}-%{release}
+%description devel-internal
+Internal development package for sub-plugins and native APIs.
```

Comment:
```
Internal development package for sub-plugins and native APIs.
```
==>
```
Development package to access internal functions of nnstreamer.
This may be used by API packages.
In most cases, custom-filter or subplugin authors do not need this internal devel package; however, if they want to access more internal functions, they may need this,
```

Reply:
Please update Ubuntu's comments as well.